src/synthesizer/particle/particles.py

```diff
@@ -227,6 +227,18 @@ def get_particle_photo_fluxes(self, filters, verbose=True):
 
         return self.particle_photo_fluxes
 
+    def integrate_particle_spectra(self):
+        """
+        Integrate any particle spectra to get integrated spectra.
+
+        This will take all spectra in self.particle_spectra and call the sum
+        method on them, populating self.spectra with the results.
+        """
+        # Loop over the particle spectra
+        for key, sed in self.particle_spectra.items():
+            # Sum the spectra
+            self.spectra[key] = sed.sum()
```

Comment:
I assume this sum is overloaded to ensure it's acting over the correct dimensions?

Reply:
Indeed, see https://github.com/flaresimulations/synthesizer/blob/main/src/synthesizer/sed.py#L145 

However, now you mention it this will only ever sum over the first dimension. We never do currently but if we had a ndim > 2 `sed` this wouldn't behave as expected. We want it to sum over all but the last axis.